### PR TITLE
Updating the readme files with the external database requirements

### PR DESCRIPTION
**Pull request by jamesrichard91**

### CODE REVIEW

File: deployment/helm/synopsys-alert/README.md [hunk 1]
```diff
@@ -500,6 +501,42 @@ For the on-premise database deployment a second Persistent Volume must be create
...
+        - DELETE
+        - EXECUTE
+    - The Alert User role should have the following privileges on all objects in the alert schema (or the schema/database when relevant):
+        - SELECT
+        - INSERT
+        - UPDATE
+        - DELETE
+        - EXECUTE
+
 - On the external database create a user to own the Alert database
```
Comment (jamesrichard91) [inline deployment/helm/synopsys-alert/README.md:540]: Are the steps from line 540 - 554 still correct? They seem  to conflict with our documented requirements.

Comment (gkillough) [reply deployment/helm/synopsys-alert/README.md:540]: @psantos1113 Is DB ownership required? It doesn't seem to be based on our liquibase. We just need to be able to perform the necessary operations in our liquibase scripts and at runtime.

File: deployment/helm/synopsys-alert/README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Helm Charts for Synopsys Alert
   - [Custom Certificates](#custom-certificates)
   - [Persistent Storage](#persistent-storage)
   - [External Postgres Database](#external-postgres-database)
+    - [External Postgres Database Requirements](#external-postgres-database-requirements)
```
Comment (psantos1113) [inline deployment/helm/synopsys-alert/README.md:18]: Appears to be an extra whitespace character here.

File: deployment/helm/synopsys-alert/README.md (hunk 1 above)
Comment (psantos1113) [reply deployment/helm/synopsys-alert/README.md:540]: They don't have to create a separate user or database for Alert.  It isn't required based on what you've added to the document.  If they have an existing database and user with the permissions, then that can be used.  The steps to create a user and a database were here to help simplify the setup process for a user of Alert and separate the Alert data from any other database data.  Ultimately it's the database administrators choice.

**Pull request merged**